FinniversKit/Sources/Components/Label/Label.swift

```diff
@@ -15,7 +15,7 @@ public class Label: UILabel {
 
     public init(
         style: Style,
-        numberOfLines: Int = 1,
+        numberOfLines: Int = 2,
```

Comment:
This should not be changed. The default should be `1`, and then uses of this can rather set `2` if needed :)